examples/run/alpaka/seeding_example_alpaka.cpp

```diff
@@ -79,7 +79,8 @@ int seq_run(const traccc::opts::track_seeding& seeding_opts,
 
     // Performance writer
     traccc::seeding_performance_writer sd_performance_writer(
-        traccc::seeding_performance_writer::config{},
+        traccc::seeding_performance_writer::config{.truth_config =
+                                                       truth_finding_opts},
```

Comment:
These are truth matching cuts in the `truth_finding_opts` or are they options for truth finding (i.e. CKF with perfect inputs)?